AddLog.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo
+echo "Log function" - Navid200
+echo
+
+# Requires two parameters.
+# The first parameter is a line, in double quotes, that will be added to the top of the log file.
+# The second parameter is the full path to the log file.
+# This is an example call to this function:   /xDrip/scripts/AddLog.sh "This is my log" /directory/LogFile
+
+if [ $# != 2 ]
+then
+  echo "Error: Incorrect number of parameters"
+  exit 5
+fi
+
+sudo rm -rf /tmp/Logs
```

Comment:
/tmp/Logs is very generic, please use something like /tmp/nsLogs

Reply:
It is supposed to be generic because this is a function.  We currently use it for ns logs as well as for auto login logs.  In the future, we can use it for any other log as well.  I didn't want to use anything specific intentionally for that reason.

Reply:
If some other program on the computer will use /tmp/Logs for anything you will be deleting their data.
The correct way to do it is to use a temporary file. See https://opensource.com/article/22/6/make-temporary-file-bash
As to not ask you to do a lot of work, I just asked you to rename it to something less generic.

Reply:
I understand now.

I have made both changes you have suggested.

Thanks